site/gulpfile.js

```diff
@@ -12,7 +13,15 @@ const concat        = require('gulp-concat');
 const download      = require('gulp-download-stream');
 const responsive    = require('gulp-responsive');
 
-const {swagList, swagImages} = require('./get-data');
+let swagList;
+let swagImages;
+
+const PRODUCTION = process.env.NODE_ENV === 'production';
+
+let manifest = {
```

Comment:
constantly const?

Reply:
I currently reassign it in the `cachebust` task: https://github.com/swapagarwal/swag-for-dev/pull/170/files/15ea821f1a5da409a223cede8e5b7e439b0e2859#diff-6eb60389fb540a5c670ae4d43af865a9R131